### ISSUE TITLE
Preludium card that allows to play a card superpassing global conditions doesn't allow to play birds

### ISSUE THREAD

**Issue by szalukisz** [terraforming-mars/terraforming-mars#5647]
URL: https://terraforming-mars.herokuapp.com/spectator?id=sca9d7cc72b73
Player color: red
Step: 5
Version: 45031d8, built at Sun, 30 Apr 2023 17:36:06 GMT
Browser: Chrome 112
![image](https://user-images.githubusercontent.com/60235401/235765533-4f210ab5-40a0-47f5-a2e6-5f71db91bac5.png)

**Comment by kberg** [comment]
I tested locally, and Birds works just fine. I can't actually tell right now, but I bet you had 9MC and some titanium. I'm going to close this, but let me know if you disagree.

**Comment by szalukisz** [comment]
I played Saturn systems (+42MC), kept 8 cards (-24MC), played merger(-42MC), played Cheung Shing Mars(+44MC), which after all give outcome **20MC**, which should be enough for playing birds which costs 10MC. What's more I played in this generation card that's sum up to 19MC (Deep Well Heating(13-2=11MC), Space Mirrors(3MC), Technology Demonstration(5MC)). So it might be some corner case, but it looks like the issue exists, maybe the level of plant production wasn't enough, but in my opinion it anyway should allow to play the card.
